src/command.ts

```diff
@@ -211,12 +213,17 @@ export class Command {
     }
 
     if (getInheritedOption(options, "debug")) {
-      logger.transports.console.level = "debug";
       options.debug = true;
     }
     if (getInheritedOption(options, "json")) {
       options.nonInteractive = true;
-      logger.transports.console.level = "none";
+    } else {
+      logger.add(
+        new winston.transports.Console({
+          level: process.env.DEBUG ? "debug" : "info",
+          format: winston.format.printf((info) => [info.message, ...(info[SPLAT] || [])].join(" ")),
```

Comment:
Can you explain this `SPLAT` stuff to a noob like me?

Reply:
JavaScript has the concept of `symbols` now which are unique identifiers that exist throughout a process, they're not strings or ints, they're just unique. This `info` object is a JS object which contains a bunch of fields keyed to symbols. When you call `logger.debug(a, b, c)` in Winston the first argument is passed as `info.message` the second, third, etc are passed as an array on the info obj keyed to `info[SPLAT]` (which may be undefined) so to mimic the only the only concat behavior, we flatten message and the splat arguments into an array and join them.